src/ui/common/form/FormLabel.js

```diff
@@ -18,13 +18,20 @@ const FormLabel = ({
     </span>
   ) : null;
 
+  const langLabelWithText = langLabelText ? (
+    <span className="label FormLabel__language-label">
+      {langLabelText}
+    </span>
+  ) : null;
+
   const fieldHelp = helpId ? (
     <FieldLevelHelp content={formattedText(helpId)} />
   ) : null;
 
   return (
     <span className="FormLabel">
       {langLabel}
+      {langLabelWithText}
```

Comment:
This could be better understandable and less error (label duplication) prone if you write:

``` {langLabel || langLabelWithText} ```

This way, you will always show one and only one labels, even if you provide 2 (in this case, precedence is given to the translated label)